server/nodejs/api/swagger.yaml

```diff
@@ -5045,22 +5045,27 @@ definitions:
         type: "string"
         description: "Alias to pay from"
       amounts:
-        type: "string"
+        type: "object"
         description: "A stringified JSON object with aliases and amounts. { \"address\"\
           :amount, ... } (numeric or string) The syscoin alias is the key, the numeric\
           \ amount (can be string) in SYS is the value."
+        properties: {}
       instantsend:
         type: "boolean"
         description: "Set to true to use InstantSend to send this transaction or false\
           \ otherwise."
       subtractfeefromamount:
-        type: "string"
+        type: "array"
```

Comment:
@quangogster @cortesa If my assumption is correct about why subtractfeefromamount is an array and not a string, we might benefit from the amounts object carrying that payload in a future release.  e.g. an "amount" has address, amount, subtractfeefromamount.  Nothing to do for this release if this works but I wouldn't mind revisiting.